### PR TITLE
Add task about checking CPU temperature

### PR DESCRIPTION
**Pull request by tomschr**
This PR contains a little task type to check CPU temperature on your system.

### CODE REVIEW

File: xml/task-check-cpu-temperature.xml [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+Adapter: ISA adapter
+Package id 0:  +43.0°C  (high = +82.0°C, crit = +100.0°C)
+Core 0:        +41.0°C  (high = +82.0°C, crit = +100.0°C)
+Core 1:        +41.0°C  (high = +82.0°C, crit = +100.0°C)
+Core 2:        +43.0°C  (high = +82.0°C, crit = +100.0°C)
+Core 3:        +41.0°C  (high = +82.0°C, crit = +100.0°C)
+Core 4:        +41.0°C  (high = +82.0°C, crit = +100.0°C)
+Core 5:        +40.0°C  (high = +82.0°C, crit = +100.0°C)</screen>
+  <para>
+   To constantly watch your temperature, run:
```
Comment (dmpop) [inline xml/task-check-cpu-temperature.xml:213]: I don't think the sensors can measure **your** temperature. :smile: 

```suggestion
   Run the command below to monitor temperature in real time:
```

File: xml/task-check-cpu-temperature.xml [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    <para> To protect your CPUs and your hardware. CPUs has a thermal protection
+     when the CPU gets too warm. This protects it from overheating. However, if
+     a CPU temperature reaches 80°C or higher, it is a warning sign that
+     something is wrong. It can be that a fan is dead, the cooling system does not
+     work properly, or the thermal paste wasn't applied correctly.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     To reduce the carbon footprint in the long term. This can help to adjust the cooling
```
[the remark this answers is not in the document]
Comment (tomschr) [reply xml/task-check-cpu-temperature.xml:90]: Right, I will adapt that sentence.

File: xml/task-check-cpu-temperature.xml (hunk 1 above)
Comment (tomschr) [reply xml/task-check-cpu-temperature.xml:213]: Haha, right.

File: xml/task-check-cpu-temperature.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (tomschr) [reply xml/task-check-cpu-temperature.xml]: Done

**Comment by taroth21** [comment]
@tomschr : A DC file is missing, could you add one please?

**Comment by tomschr** [comment]
> @tomschr : A DC file is missing, could you add one please?

@taroth21 Ahh, thanks for the hint! It's committed in a278772f8.

**Pull request merged**